package.json

```diff
@@ -10,7 +10,7 @@
     "common"
   ],
   "scripts": {
-    "test": "echo",
+    "test": "mocha test/unit-tests/*js --require test/unit-test-helpers/default_test_env.js --exit",
```

Comment:
please revert to the previous state as here: https://github.com/ooyala/html5-common/pull/94/files